utils/relationship.js

```diff
@@ -112,7 +112,11 @@ function prepareRelationshipForTemplates(entityWithConfig, relationship, generat
   }
 
   relationship.relatedField = otherEntityData.fields.find(field => field.fieldName === relationship.otherEntityField);
-  if (!relationship.relatedField && !relationship.otherEntity.embedded) {
+  if (
+    !relationship.relatedField &&
+    !relationship.otherEntity.embedded &&
+    !(relationship.otherEntity.primaryKey && relationship.otherEntity.primaryKey.composite)
+  ) {
```

Comment:
I suppose this change is related to having multiples otherEntityFields.
We don't support that in our templates. This change is specific to your blueprint.
You can extract lines 114-134 into another task in the generator and replace with your own implementation in your blueprint.

Reply:
Not really, the reason I am introducing this change, is that the blueprint throws without this line as there is no related Field. I would just like to avoid that. I can do this locally without redefining the whole function and calling  whenever it was called before.

Reply:
Unify with the condition below and drop the else throw.
So it will not be required anymore.

Reply:
@mshima done